### PR TITLE
En_Po_Sisters

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c [hunk 1]
```diff
@@ -85,114 +141,1056 @@ static DamageTable D_80B1DAA4 = {
...
+void func_80B1AE28(EnPoSisters* this) {
+    this->actionFunc = func_80B1AE3C;
+}
+
+void func_80B1AE3C(EnPoSisters* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+    s16 sp22;
+
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    sp22 = this->actor.yawTowardsPlayer - player->actor.shape.rot.y;
```
Comment (jpburnett) [inline src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c:363]:
```suggestion
    s16 sp22 = this->actor.yawTowardsPlayer - player->actor.shape.rot.y;

    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
```
Does this work, or does it mess up the stack?

**Review by jpburnett — COMMENTED** [review]
Nice job, just one minor thought

**Comment by Kenix3** [comment]
Merge pending Jenkins successful build.

**Review by hensldm — APPROVED** [review]
LTGM

**Pull request merged**